src/main/java/Tokenise.java

```diff
@@ -48,4 +48,14 @@ public Tokenise(String userInput) {
     public Boolean getIsKeyword() {
         return isKeyword;
     }
+
+    public String removeKeyword() {
+        String stringWithoutKeyword = "";
+        for (int i = 1; i < tokens.length; i++) {
+            stringWithoutKeyword = stringWithoutKeyword
```

Comment:
It is also good that the long lines and wrapped with the correct indentation as well